keyboards/atreus62/keymaps/194h/config.h

```diff
@@ -0,0 +1,8 @@
+#define ONESHOT_TIMEOUT 3000
+#define TAPPING_TERM 200
+#define PREVENT_STUCK_MODIFIERS
+#define FORCE_NKRO
+#define LEADER_TIMEOUT 1000
+#define TAPPING_TOGGLE 3
+
+#include "../../config.h"
```

Comment:
This line can be safely deleted as the build script includes all `config.h` files in the directory tree automatically.